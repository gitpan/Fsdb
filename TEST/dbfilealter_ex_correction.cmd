prog='dbfilealter'
args='-c -F D'
cmp='diff -cb '
