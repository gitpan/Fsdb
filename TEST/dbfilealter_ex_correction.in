#fsdb -F S name id test1
a student  1  80
b nice  2  70
c all  3  65
