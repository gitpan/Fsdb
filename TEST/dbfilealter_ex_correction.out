#fsdb name id test1
a_student	1	80
b_nice	2	70
c_all	3	65
#   | dbfilealter -c -F D
